Cargo.toml

```diff
@@ -9,13 +9,6 @@ homepage = "https://github.com/gltf-rs/gltf"
 readme = "README.md"
 keywords = ["glTF", "3D", "asset", "model", "scene"]
 license = "MIT/Apache-2.0"
-include = [
-    "Cargo.toml",
-    "examples/*.bin",
-    "examples/*.glb",
-    "examples/*.gltf",
-    "src/*.rs"
-]
```

Comment:
So no include/exclude? I'd have a look at the result of `cargo package` to see if there's nothing that should be excluded.